snapshot.yaml

```diff
@@ -39,3 +39,12 @@ packages:
   # https://github.com/EdutainmentLIVE/recurly
   - git: https://github.com/EdutainmentLIVE/recurly
     commit: caafdaf62918d3d41dc57f1b547507265e1ed002
+
+  # https://github.com/yesodweb/persistent
+  - git: https://github.com/yesodweb/persistent
+    commit: 0e2dc880d3b1f974a955293540b9891acba13551
+    subdirs:
+      - persistent
+      - persistent-template
+      - persistent-mongoDB
+      - persistent-postgresql
```

Comment:
I think we can use the latest Hackage versions of these dependencies rather than using a GitHub source.